src/assets/Azure.Core.Shared/DiagnosticScope.cs

```diff
@@ -207,25 +207,6 @@ public ActivityAdapter(object? activitySource, DiagnosticSource diagnosticSource
                 _activityName = activityName;
                 _kind = kind;
                 _diagnosticSourceArgs = diagnosticSourceArgs;
-
```

Comment:
FYI this change comes from build failure:

> D:\a\_work\1\s\autorest.csharp\eng\SharedCodeCheck.ps1 : Shared source files are updated. Please run eng/DownloadSharedSource.ps1

So I did run `eng/DownloadSharedSource.ps1` locally and pushed the resulting change.